docs/howto/configure-layers.md

```diff
@@ -0,0 +1,74 @@
+# Configure layers
+
+Below is an example of the current configuration format. For full details of all fields, see the [complete layer specification](../reference/layer-specification).
```

Comment:
This is a relative link inside the same docs website, instead of an external URL, so I'm keeping it for simplicity. For external URLs I'll use things like:

- [Canonical website](https://canonical.com/)
- https:/<span></span>/canonical.com/

I hope this is OK.